README.md

```diff
@@ -76,6 +78,37 @@ We recommend installing RDKit with deepchem if you use conda.
 conda install -y -c conda-forge rdkit
 ```
 
+### Latest version
+
+For using general utilites like Molnet, Featurisers, Datasets, etc, then, you install deepchem via pip.  
+
+```bash
+pip install --pre deepchem
+```
+Deepchem provides support for tensorflow, pytorch, jax and each require
+a induvidual pip Installation.
+
+For using models with tensorflow dependencies, you install using
+
+```bash
+pip install --pre deepchem[tensorflow]
+```
+For using models with torch dependencies, you install using
+
+```bash
+pip install --pre deepchem[torch]
+```
+For using models with jax dependencies, you install using
+
+```bash
+pip install --pre deepchem[jax]
+```
+If GPU support is required, then make sure its cuda packages installed and then install the NN library using the below links before installing deepchem

```

Comment:
How about: "If GPU support is required, then make sure CUDA is installed and then install the desired deep learning framework using the links below before installing deepchem"